bigtable/tests/data_integration_test.cc

```diff
@@ -169,20 +151,48 @@ TEST_F(DataIntegrationTest, TableSingleRow) {
       row_key, bigtable::SetCell(family, "c1", 1000, "V1000"),
       bigtable::SetCell(family, "c2", 2000, "V2000"),
       bigtable::SetCell(family, "c3", 3000, "V3000"));
-
   table->Apply(std::move(mutation));
-
-  auto actual = ReadRows(*table, bigtable::Filter::PassAllFilter());
-
   std::vector<bigtable::Cell> expected{
       {row_key, family, "c1", 1000, "V1000", {}},
       {row_key, family, "c2", 2000, "V2000", {}},
       {row_key, family, "c3", 3000, "V3000", {}}};
 
+  auto actual = ReadRows(*table, bigtable::Filter::PassAllFilter());
+  DeleteTable(table_name);
   CheckEqualUnordered(expected, actual);
+}
 
+TEST_F(DataIntegrationTest, TableReadRowTest) {
+  std::string const table_name = "table-read-row-test";
+  auto table = CreateTable(table_name, table_config);
+  std::string const row_key1 = "row-key-1";
+  std::string const row_key2 = "row-key-2";
+
+  std::vector<bigtable::Cell> created{
+      {row_key1, family, "c1", 1000, "v1000", {}},
+      {row_key2, family, "c2", 2000, "v2000", {}}};
+  std::vector<bigtable::Cell> expected{
+      {row_key1, family, "c1", 1000, "v1000", {}}};
+
+  CreateCells(*table, created);
+  auto row_cell = ReadRow(*table, row_key1, bigtable::Filter::PassAllFilter());
```

Comment:
Why not call `table->ReadRow()` directly here (and in the next function)?  The `ReadRow()` wrapper in the fixture does not save that much work, and actually obscures what is going on.